src/ui/Controllers/OrganisationController.cs

```diff
@@ -40,39 +42,62 @@ public async Task<IActionResult> Courses(string ucasCode)
-            {
-                var providerCourses = institutionCourses.Courses.Where(x => x.AccreditingProviderName == providerName).ToList();
-                var providerId = providerCourses.Select(x => x.AccreditingProviderId).Distinct().SingleOrDefault();//should all be the same
+            var ucasData = (await _manageApi.GetCoursesByOrganisation(ucasCode));
+            var enrichmentModel = (await _manageApi.GetEnrichmentOrganisation(ucasCode)).EnrichmentModel;
 
-                var provider = new Provider
+            var aboutAccreditingTrainingProviders = ucasData.Courses
+                .Where(x =>
+                    false == string.Equals(x.AccreditingProviderId, ucasCode, StringComparison.InvariantCultureIgnoreCase) &&
+                    false == string.IsNullOrWhiteSpace(x.AccreditingProviderId))
+                .Distinct(new AccreditingProviderIdComparer())
+                .Select(x => new TrainingProviderViewModel()
                 {
-                    ProviderId = providerId,
-                    ProviderName = providerName,
-                    Courses = providerCourses,
-                    TotalCount = providerCourses.Count
-                };
-                providers.Add(provider);
-            }
+                    InstitutionName = x.AccreditingProviderName,
+                    InstitutionCode = x.AccreditingProviderId,
+                    Description = enrichmentModel.AccreditingProviderEnrichments.FirstOrDefault(TrainingProviderMatchesProviderCourse(x))?.Description ?? ""
+                }).ToList();
 
-            return providers;
-        }
-        private async Task<TabViewModel> GetTabViewModelAsync(string ucasCode, string currentTab) {
-            var orgs = await _manageApi.GetOrganisations();
-            var organisationName = orgs.FirstOrDefault(o => ucasCode.Equals(o.UcasCode, StringComparison.InvariantCultureIgnoreCase))?.OrganisationName;
-            var result = new TabViewModel
+            var tabViewModel = await GetTabViewModelAsync(ucasCode, "about");
+
+            var model = new OrganisationViewModel
             {
-                CurrentTab = currentTab,
-                MultipleOrganisations = orgs.Count() > 1,
-                OrganisationName = organisationName,
-                UcasCode = ucasCode
+                InstitutionCode = ucasCode,
+                TabViewModel = tabViewModel,
+                TrainWithUs = enrichmentModel.TrainWithUs,
+                AboutTrainingProviders = aboutAccreditingTrainingProviders,
+                TrainWithDisability = enrichmentModel.TrainWithDisability
             };
 
-            return result;
+            return View(model);
+        }
+
+        [HttpPost]
+        [Route("{ucasCode}/about")]
+        public async Task<ActionResult> AboutPost(string ucasCode, OrganisationViewModel model)
+        {
+            var enrichmentModel = (await _manageApi.GetEnrichmentOrganisation(ucasCode)).EnrichmentModel;
```

Comment:
This GET to the API seems like an unnecessary extra round-trip. I think we can just build the model from scratch and POST it.